dev-infrastructure/templates/global-infra.bicep

```diff
@@ -34,6 +34,33 @@ resource svcParentZone 'Microsoft.Network/dnsZones@2018-05-01' = {
   location: 'global'
 }
 
+// DNS Zone Contributor: Lets SafeDnsIntApplication manage DNS zones and record sets in Azure DNS, but does not let it control who has access to them.
+// https://learn.microsoft.com/en-us/azure/role-based-access-control/built-in-roles/networking#dns-zone-contributor
+var dnsZoneContributor = subscriptionResourceId('Microsoft.Authorization/roleDefinitions/', 'befefa01-2a29-4197-83a8-272ff33ce314')
+
+//  SafeDnsIntApplication object ID use to delegate child DNS
+var safeDnsIntAppObjectId = 'c54b6bce-1cd3-4d37-bebe-aa22f4ce4fbc'
```

Comment:
please introduce a new config field for this so we can define this in config per env.
for RH you can use the same ID as we use for `aroDevopsMsiId`